packages/updating-element/README.md

```diff
@@ -0,0 +1,149 @@
+# UpdatingElement 1.0 Pre-release
+
+[![Build Status](https://github.com/polymer/lit-html/workflows/Tests/badge.svg?branch=lit-next)](https://github.com/Polymer/lit-html/actions?query=workflow%3ATests)
+[![Published on npm](https://img.shields.io/npm/v/lit-element/next-major)](https://www.npmjs.com/package/lit-html)
+[![Join our Slack](https://img.shields.io/badge/slack-join%20chat-4a154b.svg)](https://www.polymer-project.org/slack-invite)
+[![Mentioned in Awesome lit-html](https://awesome.re/mentioned-badge.svg)](https://github.com/web-padawan/awesome-lit-html)
+
+## 🚨 About this pre-release
+
+This is a major version pre-release of UpdatingElement 1.0. This package
+has been migrated out of LitElement. See issue
+[#1077](https://github.com/Polymer/lit-element/issues/1077) for more info.
+
+This pre-release is not yet feature complete or API stable.
+
+<hr>
+
+# UpdatingElement
+
+A simple low level base class for creating fast, lightweight web components.
+
+## Documentation
+
+Full documentation is available at [lit-element.polymer-project.org](https://lit-element.polymer-project.org).
+
+## Overview
+
+UpdatingElement is a low level base class for creating custom elements that
+render content to shadowRoot via a user implemented method. It adds API to help
+manage element properties and attributes. UpdatingElement reacts to changes in
+properties and calls the update method which should be implemented to update
+the rendered state of the element.
+
+```ts
+import {
+  UpdatingElement,
+  html,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+} from 'updating-element';
+
+// This decorator defines the element.
+@customElement('my-element')
+export class MyElement extends UpdatingElement {
+  // This decorator creates a property accessor that triggers rendering and
+  // an observed attribute.
+  @property()
+  mood = 'great';
+
+  static styles = css`
+    span {
+      color: green;
+    }
+  `;
+
+  update(changedProperties: PropertyValues) {
+    this.renderRoot.innerHTML = `Web Components are <span>${this.mood}</span>!`;
+  }
+}
+```
+
+```html
+<my-element mood="awesome"></my-element>
+```
+
+Note, this example uses decorators to create properties. Decorators are a proposed
+standard currently available in [TypeScript](https://www.typescriptlang.org/) or [Babel](https://babeljs.io/docs/en/babel-plugin-proposal-decorators). UpdatingElement also supports a [vanilla JavaScript method](https://lit-element.polymer-project.org/guide/properties#declare) of declaring reactive properties.
+
+## Examples
+
+- Runs in all [supported](#supported-browsers) browsers: [Glitch](https://glitch.com/edit/#!/hello-lit-element?path=index.html)
```

Comment:
These are the lit-element links... :(

Reply:
Tracked in https://github.com/Polymer/lit-html/issues/1359.